app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +393,47 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
+            <div
+              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
+            >
+              <ReportButton />
+            </div>
+            <div
+              className={classes.menuBtn}
+              onClick={menuOpen ? handleMenuClose : handleMenuOpen}
+            >
+              <MenuIcon />
+              <Avatar src={user?.avatarUrl} className={classes.avatar}>
+                {user?.avatarUrl && user.avatarUrl?.length > 0
+                  ? null
+                  : acronym(user?.name)}
+              </Avatar>
+            </div>
+          </div>
+
+          <Menu
+            open={menuOpen}
+            anchorEl={menuRef.current}
+            elevation={0}
```

Comment:
Why?

Reply:
I removed that property, but i'm not 100% sure i understand how elevation works o_O